gap/SkeletalFinSetsForCAP.gi

```diff
@@ -75,8 +75,9 @@ InstallMethod( ListOp,
         [ IsSkeletalFiniteSet, IsFunction ],
         
   function ( s, f )
-
-    return List( AsList( s ), f );
+    
+    return List( AsList( s ), x -> f(x) );
```

Comment:
Could you add a test covering this line? This method is not used anymore after your changes, but we should keep it for backwards compatibility.

Reply:
Done with 4c64b970.